README.md

```diff
@@ -49,6 +47,20 @@ To run this sample:
 6. Run the sample:
     - mvn clean compile exec:java
 
+## Use latest key vault SDK
+
+The key vault SDK package  here is **com.microsoft.azure.azure-keyvault**, if you are using the [latest](https://search.maven.org/artifact/com.azure/azure-security-keyvault-secrets) version of the key vault SDK package, please reference to the following examples:
+
+* [IdentityReadmeSamples.java](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-secrets/src/samples/java/com/azure/security/keyvault/secrets/IdentityReadmeSamples.java)- Examples to authenticate to key vault secret client
+
+    * createClientCertificateCredential: Create a secret client using cert based authentication.
+    * createDefaultAzureCredential: Create a secret client using DefaultAzureCredential.
+
+* [HelloWorld.java](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/azure-security-keyvault-secrets/src/samples/java/com/azure/security/keyvault/secrets/HelloWorld.java)  - Examples for common key vault tasks:
+
+    * Create a secret inside the keyvault
```

Comment:
keyvault -> Key Vault

Same with other usages of "key vault". It's a product name.